# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/test/gtest/TestSurfaceCache.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c image/test/gtest/TestSurfaceCache.cpp
--- a/image/test/gtest/TestSurfaceCache.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/image/test/gtest/TestSurfaceCache.cpp	Fri May 24 11:32:54 2019 +0000
@@ -11,6 +11,7 @@
 #include "ImageFactory.h"
 #include "mozilla/gfx/2D.h"
 #include "mozilla/RefPtr.h"
+#include "mozilla/StaticPrefs.h"
 #include "nsIInputStream.h"
 #include "nsString.h"
 #include "ProgressTracker.h"
@@ -43,7 +44,7 @@
   // Ensures we meet the threshold for FLAG_SYNC_DECODE_IF_FAST to do sync
   // decoding without the implications of FLAG_SYNC_DECODE.
   ASSERT_LT(length,
-            static_cast<uint64_t>(gfxPrefs::ImageMemDecodeBytesAtATime()));
+            static_cast<uint64_t>(StaticPrefs::ImageMemDecodeBytesAtATime()));
 
   // Write the data into the image.
   rv = image->OnImageDataAvailable(nullptr, nullptr, inputStream, 0,